src/plugins/kibana/public/kibana.js

```diff
@@ -12,6 +12,8 @@ var modules = require('ui/modules');
 
 var kibanaLogoUrl = require('ui/images/kibana.png');
 
+routes.start();
```

Comment:
I would prefer this be called `.enable()`, since we aren't exactly starting the router here as much as allowing it to be started later.